### DISCUSSION TITLE
Call for users!!!

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/api-testing#284] [Show and tell]
Please feel free to add it below if your project is using [api-testing](https://github.com/LinuxSuRen/api-testing) for the testing.

| Project | Description | Star | Date |
|---|---|---|---|
| [mcache-team/mcache](https://github.com/mcache-team/mcache) | Memory Cache | ![GitHub Repo stars](https://img.shields.io/github/stars/mcache-team/mcache?style=social) | June 4, 2024 |
| [halo-dev/halo](https://github.com/halo-dev/halo/pull/4892) | 建站工具 | ![GitHub Repo stars](https://img.shields.io/github/stars/halo-dev/halo?style=social) | Nov 23, 2023 |
| [apache/hertzbeat](https://github.com/apache/hertzbeat/pull/1387) | Monitoring system | ![GitHub Repo stars](https://img.shields.io/github/stars/apache/hertzbeat?style=social) | Dec 5, 2023 |

> In the adoption time order.

## On going
* [cnrancher/autok3s](https://github.com/cnrancher/autok3s/pull/647)
* [apache/incubator-answer](https://github.com/apache/incubator-answer/pull/626)

**Comment by cubxxw** [comment]
Hello @LinuxSuRen ,

I hope this message finds you well. I wanted to reach out to you regarding our two open-source projects, namely the Open IM Server (https://github.com/openimsdk/open-im-server) and Open KF (https://github.com/openimsdk/openkf). 

We are currently seeking contributors who can help us with the automation testing aspect of these projects. We believe that your expertise and skills in this field would be a valuable addition to our team. 

We understand that your time is valuable, but we would be incredibly grateful if you could spare some of it to consider collaborating with us. Your contribution would not only benefit our projects but also contribute to the open-source community as a whole. 

Thank you for your consideration, and we look forward to hearing from you soon.

Best regards,
@cubxxw

**Reply by LinuxSuRen** [reply]
hi @cubxxw , hopefully [api-testing](https://github.com/LinuxSuRen/api-testing) could help your projects' APIs always work well.
